.gitignore

```diff
@@ -30,3 +30,10 @@
 config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml
+
+# Ignore coverage directory
+coverage
+
+# Ignore IDE and rubocop files
+.idea/*
+.rubocop.yml
```

Comment:
I think it would be okay to check-in the `rubocop.yml` for this project.  It allows others that come into the project in the future (even if that never ends up happening) to know what the guidelines you were using when developing this app, and that they should follow the same.

It's your call though, I'm not sold in one direction or the other.